react/src/components/ModelCloneModal.tsx

```diff
@@ -0,0 +1,227 @@
+  const [form] = Form.useForm();
+  const painKiller = usePainKiller();
+
+  // const { data: allowed_vfolder_types } = useTanQuery({
+  //   queryKey: ['modelCloneModal', 'vfolder_allowed_types'],
+  //   queryFn: () => {
+  //     return baiClient.vfolder.list_allowed_types();
+  //   },
+  // });
+
+  const mutationToClone = useTanMutation({
+    // @ts-ignore
+    mutationFn: ({ input, name }: { input: any; name: string }) => {
+      return baiClient.vfolder.clone(input, name);
+    },
+  });
+
+  return (
+    <BAIModal
+      destroyOnClose
+      {...props}
+      okText={t('button.Clone')}
+      confirmLoading={mutationToClone.isLoading}
+      onOk={() => {
+        form
+          .validateFields()
+          .then((values) => {
+            mutationToClone.mutate(
+              {
+                input: values,
+                name: sourceFolderName,
+              },
+              {
+                onSuccess(data) {
+                  message.info({
+                    content: t('modelStore.CloneSuccess'),
+                  });
+                  props.onOk?.(data);
+                },
+                onError(error: any) {
+                  // const title = painKiller.relieve(error?.title);
+                  const messageStr = painKiller.relieve(error?.message);
+                  message.error({
+                    content: messageStr,
+                  });
+                },
+              },
+            );
+          })
+          .catch(() => {});
+      }}
+    >
+      <Flex direction="column" align="stretch" gap="sm">
+        <Alert showIcon type="info" message={t('modelStore.CloneInfo')} />
+        <Form
+          form={form}
+          layout="vertical"
+          requiredMark="optional"
+          initialValues={{
+            permission: 'rw',
+            // project: currentProject.id,
+            // type: 'user',
+            usage_mode: 'model',
+            target_name: sourceFolderName + '_1',
+            target_host: sourceFolderHost,
+          }}
+        >
+          <Form.Item label={t('data.ExistingFolderName')} required>
+            <Input value={sourceFolderName} disabled />
+          </Form.Item>
+          <Form.Item
+            name="target_name"
+            label={t('data.Foldername')}
+            rules={[
+              {
+                required: true,
+              },
+              {
+                pattern: /^[a-zA-Z0-9._-]*$/,
+                message: t('data.Allowslettersnumbersand-_dot'),
+              },
+            ]}
+          >
+            <Input autoComplete="off" />
+          </Form.Item>
+          <Form.Item
+            name="target_host"
+            label={t('data.Host')}
+            tooltip={t('data.CloningIsOnlyPossibleSameHost')}
+            rules={[
+              {
+                required: true,
+              },
+            ]}
+          >
+            <StorageSelector value={sourceFolderHost} disabled />
```

Comment:
You don't need to set value here again because there is an initialValue.
```suggestion
            <StorageSelector disabled />
```